src/openrct2/interface/Window.cpp

```diff
@@ -1082,7 +1082,7 @@ void window_zoom_out(rct_window* w, bool atCursor)
 void main_window_zoom(bool zoomIn, bool atCursor)
 {
     auto* mainWindow = window_get_main();
-    if (mainWindow != nullptr)
+    if (mainWindow == nullptr)
```

Comment:
Looks like this was caused in fdce5df6dbf6ea234663c92a2c9285f41e434ca0 , and as such it is probably present in v0.3.5.1 and would warrant a changelog entry.